### PR TITLE
fixup fault regex

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

Fix the fault regex to be accurate to new BMS and MPU methodology.
## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## Screenshots

_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window_

_If you did any manual testing (e.g., with Postman), put screenshots of the http request and before and after of the db_

_If none of this applies, you can delete this section_

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `package-lock.json` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: scylla-server/src/socket_handler.rs [hunk 1]
```diff
@@ -60,13 +60,13 @@ pub async fn socket_handler_with_metadata(
...
-    // init faults
+    // init faults, only cirtical faults, as of 3/31 can JSON
     let fault_regex_bms: Regex =
-        Regex::new(r"BMS\/Status\/F\/(.*)").expect("Could not compile regex!");
+        Regex::new(r"BMS\/Faults\/(.*)").expect("Could not compile regex!");
     let fault_regex_charger: Regex =
         Regex::new(r"Charger\/Box\/F_(.*)").expect("Could not compile regex!");
     let fault_regex_mpu: Regex =
-        Regex::new(r"MPU\/Fault\/F_(.*)").expect("Could not compile regex!");
+        Regex::new(r"MPU\/Fault\/Critical\/(.*)").expect("Could not compile regex!");
```
Comment (bracyw) [inline scylla-server/src/socket_handler.rs:67]: all faults are critical now?

**Review by bracyw — COMMENTED** [review]
Idk what's up with the merge conflict but lgtm. Just wanted to double check on faults.

File: scylla-server/src/socket_handler.rs (hunk 1 above)
Comment (jr1221) [reply scylla-server/src/socket_handler.rs:67]: right now this handler only checks critical faults.  idk if thats what we should be doing but...

**Review by bracyw — APPROVED** [review]
